components/progresscircle/index.css

```diff
@@ -17,8 +17,8 @@ governing permissions and limitations under the License.
   --spectrum-progress-circle-track-border-color: var(--spectrum-gray-300);
 
   /* circle progress fill border color */
-  --spectrum-progress-circle-fill-border-color: var(--spectrum-blue-900);
-
+  --spectrum-progress-circle-fill-border-color: var(--spectrum-accent-content-color-default);
+  
```

Comment:
yes, I think adding a new line here to match the rest of the token definitions + comment is justified. :)

Reply:
Is this the correct token because it is an alias and [accent-color-900](https://github.com/adobe/spectrum-tokens/blob/beta/src/semantic-color-palette.json#L82) is global?